scripts/apps/authoring-react/data-layer.ts

```diff
@@ -122,6 +123,28 @@ function getArticleContentProfile<T>(item: IArticle, fieldsAdapter: IFieldsAdapt
             }
         }
 
+        // TODO: write an upgrade script and remove hardcoding
+        // after angular based authoring is removed from the codebase
+        if (item.type === 'picture') {
+            const fieldConfig: IEditor3Config = {
+                editorFormat: [],
+                minLength: fakeScope.schema?.minlength,
+                maxLength: fakeScope.schema?.maxlength,
+                cleanPastedHtml: fakeScope.editor?.cleanPastedHTML,
+                singleLine: true,
+                disallowedCharacters: [],
+            };
+
+            const description_text: IAuthoringFieldV2 = {
+                id: 'description_text',
```

Comment:
Does it get written to the correct place? I'd think that without an adapter, it would get saved to IArticle>extra>description_text, when we want IArticle>description_text, no?

Reply:
You're right, we'll need an adapter to write it to `article.description_text`